tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/action/CreateDBConnectAction.java

```diff
@@ -347,21 +347,23 @@ public void addDatabaseConnectionProfile( String connectionName, String driverPa
                 DriverManager.getInstance().createNewDriverInstance(
                     driverTemplate, uniqueDriverInstanceName, driverPath, driverClass );
 
-            final String vendor = driverInstance.getProperty( IJDBCDriverDefinitionConstants.DATABASE_VENDOR_PROP_ID );
-            final String uniqueConnectionProfileName = generateUniqueConnectionProfileName( connectionName + " " + vendor ); //$NON-NLS-1$
-
-            final Properties connectionProfileProperties = driverInstance.getPropertySet().getBaseProperties();
-
-            connectionProfileProperties.setProperty(
-                ConnectionProfileConstants.PROP_DRIVER_DEFINITION_ID, driverInstance.getId() );
-            connectionProfileProperties.setProperty(
-                IJDBCDriverDefinitionConstants.DATABASE_NAME_PROP_ID, getDatabaseName( connectionUrl ) );
-            connectionProfileProperties.setProperty( IJDBCDriverDefinitionConstants.USERNAME_PROP_ID, userName );
-            connectionProfileProperties.setProperty( IJDBCDriverDefinitionConstants.PASSWORD_PROP_ID, password );
-            connectionProfileProperties.setProperty( IJDBCDriverDefinitionConstants.URL_PROP_ID, connectionUrl );
-
-            ProfileManager.getInstance().createProfile(
-                uniqueConnectionProfileName, connectionDesc, providerId, connectionProfileProperties, "", false ); //$NON-NLS-1$
+			if (driverInstance != null) {
```

Comment:
If driver instance is null, we need to let the user know, instead of just silently not adding a database connection profile.

Reply:
Yeah,I will deal with it properly